manifests/m/magic-wormhole/magic-wormhole/0.12.0/magic-wormhole.magic-wormhole.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: magic-wormhole.magic-wormhole
```

```suggestion
PackageIdentifier: aquacash5.magic-wormhole
```
For releases through github, its best to use the Owner.Package format for package identifiers

---

The package I am uploading is just a repackaging I made of the project for Chocolatey. The real owner of the project is correctly attributed https://github.com/magic-wormhole/magic-wormhole